mitiq/tests/test_folding_cirq.py

```diff
@@ -543,25 +543,24 @@ def test_fold_random_no_repeats():
     """Tests folding at random to ensure that no gates are folded twice and folded gates
     are not folded again.
     """
-    # Test circuit
-    # 0: ───H───@───@───
-    #           │   │
-    # 1: ───H───X───@───
-    #               │
-    # 2: ───H───T───X───
-    qreg = LineQubit.range(3)
+    # Test circuit:
+    # 0: ───H───@───Y───@───
+    #           │       │
+    # 1: ───────X───X───@───
```

Comment:
```suggestion
    # 1: ───H───X───X───Z───
```

Reply:
Do you mean change the actual circuit? Or just the comment? Note that this is default behavior for printing circuits in Cirq.

I wouldn't put another H on qubit 1 because I want every gate to occur only once for this test.

Reply:
Sorry, I was wrong about H.
However I think that the last gate should be @--Z  and not @-@.

Reply:
@--@ and @--Z are the same for the Cirq circuit drawer.